integration_tests/ibc_v18.py

```
@@ -0,0 +1,58 @@
+import requests
+
+
+def preupgrade_ibc_snapshot(port):
+    """
+    Snapshot IBC client states before the v1.8 upgrade.
+
+    Returns a dict passed to check_ibc_client_states to verify all
+    pre-upgrade clients survive the upgrade intact.
+    """
+    url = f"http://127.0.0.1:{port}/ibc/core/client/v1/client_states"
+    resp = requests.get(url)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)